cmd/license-header-checker/main.go

```diff
@@ -91,11 +120,16 @@ func processFiles(options config.Options) {
 	}
 
 	wg.Wait()
+
 	elapsed := time.Since(start)
-	printResults(files, processedFiles, elapsed.Milliseconds(), options)
+	printResults(files, processedFiles, skipped, licenseOk, licenseAdded, licenseReplaced, elapsed.Milliseconds(), options)
 }
 
-func processFile(path string, license string, options config.Options) bool {
+func processFile(path string, license string, options config.Options) Operation {
+	green := color.FgGreen.Render
+	yellow := color.FgYellow.Render
+	red := color.FgRed.Render
+
 	data, err := ioutil.ReadFile(path)
 	if err != nil {
 		panic(err)
```

Comment:
**processFile** must return the error, **panic** will stop all in-progress operations and break the process only because 1 file fails

Reply:
👍